app/recipe/tests/test_ingredients_api.py

```diff
@@ -0,0 +1,70 @@
+from django.urls import reverse
+from django.test import TestCase
+
+from rest_framework import status
+from rest_framework.test import APIClient
+
+from core.models import Ingredient, Recipe
+
+from recipe.serializers import IngredientSerializer
+
+
+INGREDIENTS_URL = reverse('recipe:ingredient-list')
+
+
+class IngredientsApiTests(TestCase):
+    """Tests ingredients api"""
+
+    def setUp(self):
+        self.client = APIClient()
+
+    def test_retrieve_ingredient_list(self):
+        """Test retrieving a list of the ingredients"""
+        Ingredient.objects.create(name='Kale')
```

Comment:
It goes beyond the scope of this exercise, but just so that you know it exists: https://factoryboy.readthedocs.io/en/stable/orms.html

We use this in our code to generate dummy data.

Reply:
I didn't know it, it looks much easier and a cleaner way to do it. Thanks :D